packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -180,6 +180,8 @@ export const messages = {
   force_project_create_analytics_template_display_text: 'Analytics',
   force_project_create_empty_template: 'Empty project template',
   force_project_create_analytics_template: 'Analytics project template',
+  force_project_create_functions_template_display_text: 'Functions',
```

Comment:
@ekapner @danyusf text for the new option in "SFDX: Create Project".

Reply:
Fine with me.